### DISCUSSION TITLE
[Request] Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by tetrisgm** [linuxserver/Heimdall#706] [Ideas]
I use Heimdall as a launcher (I'm assuming most people do). It would be nice to have the ability to open my shortcuts with a keystroke rather than clicking on the links.

For example: press 1 for the first shortcut, 2 for the next, etc.
Alternatively, doing user defined shortcuts would be nice.

In my opinion, the ultimate way of doing this would be via the text input field on the page.
Having a way to start Heimdall with the field selected by default, and typing either 1, or maybe a command like g 1 (for go to 1) rather than performing a search would be very helpful!

**Comment by aptonline** [comment]
Also for accessibility reasons the ability to 'tab' or use the arrow keys to select apps and then 'launch' with the space bar or return would be a great addition to an already amazing tool.
